src/js/components/Tip/stories/typescript/Simple.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { useRef } from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { grommet, Box, Button, Grommet, Text } from 'grommet';
+import { Tip } from '../../Tip';
+
+const Example = () => {
+  const ref = useRef();
+  return (
+    <Grommet full theme={grommet}>
+      <Box align="center" justify="center" fill>
+        <Tip content="tooltip">
```

Comment:
I think the grommet theme should have a Tip content styled to look a bit more grommet-y. This story, which I think should work in the 80% case, seems more "plain".

Reply:
Check out base.js to see the default values, and let me know if you'd like to tweak any styles.
I've also added a separate `plain` option.

Reply:
One thing to consider is that "background-contrast" has opacity, so the default behavior will be to have opacity on the tooltip.